bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,330 @@
+
+The creation process requires your previously generated BLS key, 
+which should be located at `<path>/config/priv_validator_key.json`, 
+where `<path>` is the `--home` directory you specified when setting up your node.
+
+To create your validator, run the following command:
+
+```shell 
+babylond tx checkpointing create-validator \
+    ./<home-path>/config/validator.json \
+    --chain-id bbn-test-5 \
+    --gas "auto" \
+    --gas-adjustment 1.5 \
+    --gas-prices "0.005ubbn" \
+    --from <your-key-name>
+    --keyring-backend <keyring-backend>
+```
+
+Parameters:
+- `--chain-id`: The network identifier
+- `--gas`: Set to "auto" to automatically calculate the gas needed
+- `--gas-adjustment`: A multiplier for the estimated gas 
+- `--gas-prices`: Transaction fee in ubbn per unit of gas
+- `--from`: Your key name or address that will sign and pay for this transaction
+
+Upon successful creation, you'll be asked to approve the transaction. 
+Once approved, you'll receive a transaction hash and your validator
+operator address 
+(e.g., `bbnvaloper1qh8444k43spt6m8ernm8phxr332k85teavxmuq`).
+
+### 5.1 Verifying Validator Setup
+
+To verify your validator setup, you can use the following steps:
+
+First, get your validator's operator address using your Babylon address: 
+
+```shell
+babylond keys show <your-key-name> --address --bech val --home <path> --keyring-backend <keyring-backend>
+```
+
+For example, for the address we used above is `bbn1qh8444k43spt6m8ernm8phxr332k85teavxmuq`, 
+the operator address is `bbnvaloper1qh8444k43spt6m8ernm8phxr332k85teavxmuq`. 
+
+Next, inspect your validator's details: 
+
+```shell 
+babylond query staking validator <validator-operator-address>
+```
+
+The output should return the selected validator's configuration.
+
+```yaml 
+validator:
+  commission:
+    rates:
+      current: "100000000000000000" 
+      max: "1000000000000000000" 
+      max_change: "10000000000000000"
+  description:
+    moniker: "my-validator" 
+    website: "https://myweb.site" 
+    security_contact: "my-validator0@gmail.com"
+  status: 1 
+  tokens: "100"
+```
+
+Usually when first creating a validator, the immediate status will be 
+`BOND_STATUS_UNBONDED`. To see your validators status change you will need to 
+wait for the epoch to end.
+
+```shell 
+babylond query staking validators
+```
+
+### 5.2 Understanding Validator Status
+
+Your validator enters the active set based on two conditions: 
+1. The completion of the current epoch (a network-wide time period for 
+coordinating activities) 
+2. Having sufficient stake to qualify for the active set.
+
+When active, your status will show as `BOND_STATUS_BONDED`.
+
+The other status codes are:
+
+```shell
+BOND_STATUS_UNSPECIFIED = 0
+BOND_STATUS_UNBONDED = 1
+BOND_STATUS_UNBONDING = 2
+BOND_STATUS_BONDED = 3
+```
+
+### 5.3 Managing Your Validator
+
+For validator operations, use the standard Cosmos SDK staking module commands:
```

Comment:
This is not right. The `epoching` module should be used for staking commands. We should heavily emphasize here that (1) `x/epoching` module wraps the commands of `x/staking` and (2) all staking related transactions will take into effect at the end of the epoch.